tools/nvme.sh

```diff
@@ -46,7 +74,12 @@ case $1 in
 
                 declare -i SECTORS=$SIZE/4096
                 echo "Creating Namespaces on ${PDFIDS[$INDEX]} with size ${SIZE}"
-                switchtec-nvme create-ns ${PDFIDS[$INDEX]}@$SWITCH --nsze=$SECTORS --ncap=$SECTORS --block-size=4096
+                if [ -z $TIMER ]
```

Comment:
I think if you can just use string substitution as done in rabbit-s.sh

```bash
#!/bin/bash

echo "no time"
TIME=""
$TIME sleep 1

echo "with time"
TIME="time"
$TIME sleep 1
```
```
$ ./test.sh 
no time
with time
        1.00 real         0.00 user         0.00 sys
```

Reply:
String substitution has issues, but as you suggested in another conversation, alias works.